### ISSUE TITLE
Add hv.help(hv.<element>)  to all reference notebooks

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/holoviews#5423]
## Motivation

HoloViews can be so hard to navigate and understand. Many users have over time requested some web page where they can easily browse the parameters.

My experience is that every time I want to make a specific plot, the reference documentation is what I go to first. The available parameters and options should be described there.

It would make a world of a difference.

![image](https://user-images.githubusercontent.com/42288570/188300635-4de3c476-ff43-4914-b9db-bf5e72623ea9.png)

## Additional Context

- `hv.help` should be fixed for this to be truly helpful. See https://github.com/holoviz/holoviews/issues/5421

**Comment by jbednar** [comment]
I completely agree, especially if we can quickly create an HTML version of the help. It's such a glaring omission!
